### DISCUSSION TITLE
Jam Matchmaking (was Ludum Tinder)

### DISCUSSION THREAD

**Discussion by Cerno-b** [ludumdare/ludumdare#2022] [Ideas]
How about creating a marketplace for people who want to participate in the Jam but have no team? 

It could be a part of the User Profile where people can offer their skills via ticking checkboxes or adding tags, like 
- looking for Jam partners (LD39)
- can do art (pixel art, 3d modeling)
- can do coding (C++, Unity)
- can compose (retro, classical)

Teams can then search for specific users to fill their ranks. It would be a cool way to attract new participants that so far haven't dared enter because they were lacking a specific set of skills.

Of course that would require a working communication system within LD.

**Comment by local-minimum** [comment]
I think that would end up on the jammer.bio site, there's a bit about it on https://jammer.vg/

**Comment by zwrawr** [comment]
Weird that this is the second issue that references tinder.

#74 would provide the 'good at art' bit
#797 would be a good place for a `Need a team` thing

**Comment by local-minimum** [comment]
Should the team selection be swipe left/right to select who to invite?

**Comment by Cerno-b** [comment]
Ah, crap, didn't know this feature was already planned.

Swipe left please.

**Comment by mikekasprzak** [comment]
Longer term yeah, matchmaking is a big issue I want us to tackle.
